.github/workflows/continuous-integration-workflow.yml

```diff
@@ -10,7 +10,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        cmd: ['go_core_tests']
+        cmd: ['go_core_tests', 'go_core_race_tests']
```

Comment:
I don't think we want to turn this back on yet, it needs some work first. Can we remove it and keep the other stuff?

Reply:
Yeah for sure, just added it in to test if logs updated properly